src/workloads/scale/ManyUpdate.yml

```diff
@@ -0,0 +1,72 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-replication"
+Description: |
```

Comment:
Could you add some "Keywords" also? https://github.com/mongodb/genny/blob/2ae299767d0c870a57d1e80301616ed0309e7cac/docs/using.md#L173

We don't enforce anything on it yet, but we hope to use the keywords for workload discovery. A search of the code base will show other keywords people have used. Try to not add duplicate variation of keywords, but if you do we will fix them when we start linting the keywords.

Reply:
Added some keywords

Reply:
Thanks! Could you add one or two more for the idea that this is meant to test oplog processing overhead? I'm bad with names, but could imagine oplogOverhead primaryOverhead or something along those lines. I'm sure we do not have any such keywords now, so that's all up to you.